airbyte-workers/Dockerfile

```diff
@@ -20,5 +20,11 @@ WORKDIR /app
 # Move worker app
 ADD bin/${APPLICATION}-${VERSION}.tar /app
 
+# Grab well-known types file
+RUN wget https://github.com/mikefarah/yq/releases/download/v4.30.4/yq_linux_${DOCKER_BUILD_ARCH}.tar.gz -O - | tar xz \
+        && mv yq_linux_${DOCKER_BUILD_ARCH} /usr/bin/yq
+COPY well_known_types.yaml /app
+RUN yq -o json /app/well_known_types.yaml > /app/WellKnownTypes.json
```

Comment:
It looks like we only need the json. Can we have this as a gradle task rather than having file transformation in the Dockerfile?

Reply:
done